traffic-participants.rst

```diff
@@ -0,0 +1,96 @@
+Modeling traffic participants
+==============================
+
+Traffic participants
+--------------------
+
+A traffic participant is an element of the simulated world, which can change its position and orientation during simulation time.
```

Comment:
"... which can change its state (e.g. position and orientation) ..." would be more precise I think.

Reply:
- [ ] Make first sentence more precise.